woocommerce/woocommerce
# Float the dismiss button instead of absolute-position it.

To prevent overlaps with content/text of the message.

### All Submissions:

* [ ] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)? (not really: it uses magic numbers, restates default property & value combinations, also I could not figure out what order the properties should be in, etc. but happy to update when I receive feedback)





### Changes proposed in this Pull Request:



Closes #19793 .

### How to test the changes in this Pull Request:

1. Turn off HTTPS
2. Deactivate and reactivate WooCommerce.
3. Notice the overlapping text and Dismiss button:
![overlap_1](https://user-images.githubusercontent.com/2207451/39131129-03db8e98-470f-11e8-8e50-5b74d7e0bb29.png)
4. WC > Settings > Products
5. Notice the overlapping text and Dismiss button:
![overlap_2](https://user-images.githubusercontent.com/2207451/39131159-19ce970e-470f-11e8-9300-c0322628c382.png)
![overlap_rtl_2](https://user-images.githubusercontent.com/2207451/39131301-6a4891c6-470f-11e8-8820-3b5c88e63a31.png)
6. Switch to new branch.
7. Repeat steps 2.-5. and observe no overlapping:
![non_overlap_2](https://user-images.githubusercontent.com/2207451/39131433-ac904434-470f-11e8-9e82-b102f3ffd5d7.png)
![non_overlap_2_rtl](https://user-images.githubusercontent.com/2207451/39131437-af11c8ea-470f-11e8-8d8d-1f3c131fb6bc.png)


### Other information:

* [x] Have you successfully ran tests with your changes locally? (only manual and phpunit)



### Changelog entry

> Switched admin notice dismiss button from absolute position to float to prevent overlap between text/content of the message and the `Dismiss` button.
